mkdocs.yml

```diff
@@ -49,6 +49,7 @@ nav:
   - OVERVIEW:
       - "Readme": "index.md"
       - "Getting started": "tutorials/getting-started.ipynb"
+      - "Work with us!": "work-with-us.md"
```

Comment:
@MattF-NSIDC I love the idea of placing this page front and center under the Overview. I'm also wondering whether it makes sense to connect it with the [Contributing](https://earthaccess.readthedocs.io/en/latest/CONTRIBUTING/) guide somehow? I get a little stuck sometimes with the navigation in our readthedocs since we have a more technical-focused TOC on the righthand side of the page.  But again I think this is a great spot for it.

Reply:
I'm also reminded of @andypbarrett 's PR #498 which is overhauling the existing structure. Let's coordinate on the best placement for this based on Andy's analysis.

Reply:
Perhaps `work-with-us.md` becomes a sub page of https://earthaccess--498.org.readthedocs.build/en/498/ (the default Welcome page)?

Reply:
> Perhaps `work-with-us.md` becomes a sub page of https://earthaccess--498.org.readthedocs.build/en/498/ (the default Welcome page)?

Like a section on that page? For my personal taste, that page is getting a bit long and I would really like to break it up a bit, e.g. separate top-level items (or expanded sub-items of "welcome") on the sidebar. E.g.:

* Welcome: Prose introducing the project
    * Quick start: Installing & Getting data in 3 steps
    * Work with us: Contributors and prose
        * Contributing guide
        * Collaboration events (?)
    * Talks & external resources (perhaps this could even be moved in to reference or if we want, a new "explanation" section for the 4th diataxis quadrant... but maybe that's a separate topic :smile:)
* HOW-TO
* TUTORIALS
* USER REFERENCE
* EXPLANATION

Let's chat more tomorrow :)

Reply:
>I'm also wondering whether it makes sense to connect it with the [Contributing](https://earthaccess.readthedocs.io/en/latest/CONTRIBUTING/) guide somehow? I get a little stuck sometimes with the navigation in our readthedocs since we have a more technical-focused TOC on the righthand side of the page.

Yes, this makes a lot of sense to me also. I created #521 earlier today because I was thinking about how hard-to-navigate that page was getting :)

Reply:
Rebased on to #498